### PR TITLE
(PA-5329) Add macOS 13 (ARM) platform definition

### PR DESCRIPTION
**Pull request by skyamgarp**
Please add all notable changes to the "Unreleased" section of the CHANGELOG.

### CODE REVIEW

**Comment by mhashizume** [comment]
@skyamgarp could you update the CHANGELOG as well please? Thank you!

**Review by e-gris — COMMENTED** [review]
Please add a CHANGELOG entry to this PR. 

LGTM otherwise.

**Comment by mhashizume** [comment]
I've merged https://github.com/puppetlabs/vanagon/pull/759, so you should be able to remove most of a4914096340bf1b671de9413d383292d01429683

File: lib/vanagon/platform/defaults/osx-13-arm64.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+platform 'osx-13-arm64' do |plat|
+    plat.brew '/opt/homebrew/bin/brew'
+    plat.servicetype 'launchd'
+    plat.servicedir '/Library/LaunchDaemons'
+    plat.codename 'ventura'
+
+    plat.provision_with 'export HOMEBREW_NO_EMOJI=true'
+    plat.provision_with 'export HOMEBREW_VERBOSE=true'
+    plat.provision_with "export HOMEBREW_NO_ANALYTICS=1"
```
Comment (e-gris) [inline lib/vanagon/platform/defaults/osx-13-arm64.rb:9]: Trivial/nitpicky: use single quotes.

**Review by e-gris — APPROVED** [review]
Feel free to ignore or heed my one nitpicky comment

**Pull request merged**